be/src/runtime/runtime_state.h

```diff
@@ -283,7 +283,7 @@ class RuntimeState {
 
     Status _build_global_dict(const GlobalDictLists& global_dict_list, vectorized::GlobalDictMaps* result);
 
-    static const int DEFAULT_BATCH_SIZE = 2048;
+    static const int DEFAULT_CHUNK_SIZE = 2048;
```

Comment:
default = 2048 ?

Reply:
Yes，This is consistent with the original semantics.